### PR TITLE
feat: improve monetization docs

### PR DESCRIPTION
**Pull request by mhamas**
Small tweaks in Payouts and QA docs

### CODE REVIEW

**Review by Jkuzz — APPROVED** [review]

File: sources/academy/platform/get_most_of_actors/monetizing_your_actor.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ At the end of each month, we will then add up all rental fees collected in that
 
 ## Pay-per-result pricing model {#pay-per-result-pricing-model}
 
-If you make your Actor pay-per-result instead, you will set the price per 1,000 results, and users will be charged for your Actor solely based on the number of results your Actor produces. That's your **revenue**. The underlying [platform usage](https://docs.apify.com/platform/actors/running/usage-and-resources) the Actor generates is your **cost**. Your profit which is then paid to you is computed as 80% of the revenue minus the costs, i.e. `(0.8 * revenue) - costs = profit`. This profit is also paid out monthly.
+If you make your Actor pay-per-result instead, you will set the price per 1,000 results, and users will be charged for your Actor solely based on the number of results your Actor produces. That's your **revenue**. The underlying [platform usage](https://docs.apify.com/platform/actors/running/usage-and-resources) the Actor generates, using the free plan unit pricing, is your **cost**. Your profit which is then paid to you is computed as 80% of the revenue minus the costs, i.e. `(0.8 * revenue) - costs = profit`. This profit is also paid out monthly.
```
Comment (TC-MO) [inline sources/academy/platform/get_most_of_actors/monetizing_your_actor.md:34]:
```suggestion
If you make your Actor pay-per-result instead, you will set the price per 1,000 results, and users will be charged for your Actor solely based on the number of results your Actor produces. That's your _revenue_. The underlying [platform usage](https://docs.apify.com/platform/actors/running/usage-and-resources) the Actor generates, using the free plan unit pricing, is your _cost_. Your profit which is then paid to you is computed as 80% of the revenue minus the costs, i.e. `(0.8 * revenue) - costs = profit`. This profit is also paid out monthly.
```

Comment (TC-MO) [reply sources/academy/platform/get_most_of_actors/monetizing_your_actor.md:34]: as per style guide bold in prose should be used only to highlight UI elements

File: sources/academy/platform/get_most_of_actors/monetizing_your_actor.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Also, note that you are allowed to adjust the monetization settings of the Actor
...
+Payout invoices are created automatically on 14th of the new month. You will be notified and can access the new payout invoice, and any historical invoices, in the [**Settings > Payouts**](https://console.apify.com/account/payouts) section in Apify Console. Once the new invoice is ready, you will need to review and approve it within a week in order for the payment to be processed. Note that if you do not approve the invoice, or let us know of any discrepancies, by the 20th of the month, the invoice will be auto-approved by the system on the 21st. If you have multiple Actors, the total profit is the sum of individual profits, and any losses and gains cancel out.
```
Comment (TC-MO) [inline sources/academy/platform/get_most_of_actors/monetizing_your_actor.md:77]:
```suggestion
Payout invoices are created automatically on 14th of each month. You will be notified and can access the new payout invoice, and any historical invoices, in the [**Settings > Payouts**](https://console.apify.com/account/payouts) section in Apify Console. Once the new invoice is ready, you will need to review and approve it within a week in order for the payment to be processed. Note that if you do not approve the invoice, or let us know of any discrepancies, by the 20th of the month, the invoice will be auto-approved by the system on the 21st. If you have multiple Actors, the total profit is the sum of individual profits, and any losses and gains cancel out.
```

I think this makes it a little bit less ambiguous

**Review by TC-MO — CHANGES_REQUESTED** [review]

**Comment by mhamas** [comment]
@TC-MO applied suggestions, can you please re-review?

**Review by TC-MO — APPROVED** [review]

**Comment by mhamas** [comment]
@TC-MO can you please check the lint errors? They seem unrelated, but can you confirm I can merge this? Thx

**Comment by TC-MO** [comment]
They are unrelated to the changes but are found in the file that has been changed. In these cases I'd prefer not to leave any mdlint issues living in docs. This should take care of them.

File: sources/platform/actors/publishing/testing.mdx [hunk 3]
```diff
@@ -10,27 +10,30 @@ sidebar_position: 3
...
+
 The best course of action is to fix the Actor and rebuild it. The automatic testing system will pick this up within 24 hours
 and mark it as healthy. In some cases, your Actor might break because of issues with the target website. In such a case, if your Actor
 passes the majority of test runs in the next 7 days, it will be marked as healthy automatically.
 
 ## What if my Actor cannot comply with the test logic?
 
 Actors that require some sort of authentication will always fail the tests despite being fully functional. Also, some Actors inherently run for longer than 5 minutes.
-If that's the case with your Actor, please contact support at **support@apify.com** and explain your specific use case that justifies why the Actor should be excluded from the automated tests.
+If that's the case with your Actor, please contact support at [support@apify.com](mailto:support@apify.com) and explain your specific use case that justifies why the Actor should be excluded from the automated tests.
```
Comment (honzajavorek) [inline sources/platform/actors/publishing/testing.mdx:36]: @TC-MO Docusaurus seems to turn email addresses to links automatically. I just tried it locally and it works.

Screenshot 2024-05-20 at 11 05 26

Screenshot 2024-05-20 at 11 05 51

**Pull request merged**